src/ui/public/kbn_top_nav/kbn_top_nav.js

```diff
@@ -1,6 +1,7 @@
 import _ from 'lodash';
 import 'ui/watch_multi';
 import angular from 'angular';
+import chrome from 'ui/chrome';
```

Comment:
Good catch, I was originally exposing it on scope, but since I wasn't able to create a childscope for this directive I didn't want to leak it out and ended up exposing the one necessary method via the `KbnTopNavController`